src/main/java/com/scaleunlimited/flinkcrawler/tools/CrawlTopology.java

```diff
@@ -77,104 +76,111 @@
  * 
  * State Checkpoints in Iterative Jobs
  * 
- * Flink currently only provides processing guarantees for jobs without iterations. Enabling checkpointing on an
```

Comment:
I think reformatting files is needed, but it makes a review almost impossible. Maybe we should do a single pass to reformat everything, after verifying that nobody has an active branch that will need merging.

Reply:
Agreed. I'm even willing to merge such changes into any of my active branches, but it would be ideal to choose a branching nadir for this.